qiskit/pulse/pulse.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=invalid-name,missing-docstring,missing-param-doc
+
+"""
+Pulse envelope generation.
+"""
+
+import warnings
+from inspect import signature
+
+import numpy as np
+
+from qiskit.exceptions import QiskitError
+from qiskit.tools.visualization._pulse_visualization import pulse_drawer
```

Comment:
I think this should be imported inside the `plot()` method. Otherwise it will raise an error even when the user just wants to build a pulse, and not visualize it (if they do not have matplotlib).